### PR TITLE
Code Quality: Refactor Dialog pages

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Description

Added detailed XAML comment and refactored code-behind codes in `Files.App.Dialogs` namespace classes.

### Motivation and Context

One of the plan to add XAML comments.

### PR checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- [x] Did you build the app and test your changes?
- [x] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

_None_

### CODE REVIEW

File: src/Files.App/Dialogs/FilesystemOperationDialog.xaml [hunk 1]
```diff
@@ -183,34 +190,37 @@
...
 
+			<!--  (Divider)  -->
 			<Border
-				x:Name="SeparatorLine"
 				Height="1"
 				HorizontalAlignment="Stretch"
-				Background="{ThemeResource SystemBaseLowColor}"
-				Opacity="0.8"
-				Visibility="{x:Bind ViewModel.FileSystemDialogMode.ConflictsExist}" />
+				Background="{ThemeResource DividerStrokeColorDefaultBrush}"
```
Comment (Jay-o-Way) [inline src/Files.App/Dialogs/FilesystemOperationDialog.xaml:207]: Background or Border? I suppuse it won't matter much..

**Comment by Jay-o-Way** [comment]
I see you applied the rule-of-4, good job! 😄

File: src/Files.App/Dialogs/FilesystemOperationDialog.xaml (hunk 1 above)
Comment (0x5bfa) [reply src/Files.App/Dialogs/FilesystemOperationDialog.xaml:207]: What do you mean?

I changed separator control from `MenuflyoutSeparator` to `Border` and background from `SystemBaseLowColor` to `DividerStrokeColorDefaultBrush`

**Comment by 0x5bfa** [comment]
@yair100 Ready for review.

**Comment by QuaintMako** [comment]
Haven't seen anything suspicious in the code. Another read is more than welcome though, I am not fluent in `.xmal`.

**Pull request closed without merging**